### ISSUE TITLE
Grav / WordPress Preset Setting Enable / Disable

### ISSUE THREAD

**Issue by simmonsr** [gantry/gantry5#1863]
Like we have in Joomla with Gantry5 - Presets plugin, can we have setting in Platform Settings for Grav / WordPress that disables Presets (?presets=preset1)

**Comment by simmonsr** [comment]
If we end up implementing this, I would recommend having the default be 'disabled'.

**Comment by mahagr** [comment]
Try `?reset-settings=1`, should work in all platforms.

**Comment by simmonsr** [comment]
Yes, I am aware of ?reset-settings, but if I have preset plugin enabled in Joomla instance and go to preset URL in Joomla, it can effect Grav / WordPress instances (kind of odd really). Using ?reset-settings in Grav instance doesn't help. I have to do ?reset-settings in Joomla to clear it out for Grav. Would just be nice if WP / Grav had preset option that could be enabled / disabled (default disabled).

**Comment by simmonsr** [comment]
I really think this setting should be implemented for Grav / WP. If you think about it, if a user grabs a skeleton / RL and selects one of the Styles URL (?presets=preset1) and then tries to use the Style panel it's not going to show up until cookie is cleared (or enters ?reset-settings).

**Comment by w00fz** [comment]
I thought we disabled presets plugin by default with RL and skeletons?

**Comment by simmonsr** [comment]
Yes for Joomla, but there is no preset option in RL / skeletons for WP / Grav.

**Comment by mahagr** [comment]
There is no presets plugin for WP or Grav; in both it is integrated to Gantry itself.

**Comment by simmonsr** [comment]
Yah, I'm not asking for a plugin. Just a 'Platform Setting' if possible. :)
